### ISSUE TITLE
WebTorrent: `torrent.swarm` is deprecated.

### ISSUE THREAD

**Issue by hisener** [feross/webtorrent#765]
* WebTorrent version: 1.1.2 (webtorrent 0.91.0)
* Node.js version: v5.11.0
* Browser name/version (if using WebTorrent in the browser):

webtorrent-cli on npm 3.8.6 on Ubuntu 16.04 LTS

```
$ webtorrent --vlc magnet_uri

WebTorrent: `torrent.swarm` is deprecated. Use `torrent` directly instead.

UNEXPECTED ERROR: If this is a bug in WebTorrent, report it!
OPEN AN ISSUE: https://github.com/feross/webtorrent/issues

DEBUG INFO: webtorrent-cli 1.1.2, webtorrent 0.91.0, node v5.11.0, linux x64, exit 1
/usr/local/lib/node_modules/webtorrent-cli/bin/cmd.js:488
    var unchoked = torrent.swarm.wires.filter(function (wire) {
                                ^

TypeError: Cannot read property 'wires' of undefined
    at Timeout.draw [as _repeat] (/usr/local/lib/node_modules/webtorrent-cli/bin/cmd.js:488:33)
    at Timeout.wrapper [as _onTimeout] (timers.js:408:11)
    at Timer.unrefdHandle (timers.js:445:14)
```
**Update:**
I have deleted "swarm"s from cmd.js it works but when torrent finished this error appeared.

```
torrent downloaded successfully from 2/3 peers in 482s!

webtorrent is exiting...

UNEXPECTED ERROR: If this is a bug in WebTorrent, report it!
OPEN AN ISSUE: https://github.com/feross/webtorrent/issues

DEBUG INFO: webtorrent-cli 1.1.2, webtorrent 0.91.0, node v5.11.0, linux x64, exit 1
/usr/local/lib/node_modules/webtorrent-cli/node_modules/webtorrent/lib/torrent.js:1471
    if (self.discovery.tracker) self.discovery.tracker.complete()
                      ^

TypeError: Cannot read property 'tracker' of null
    at Torrent._checkDone (/usr/local/lib/node_modules/webtorrent-cli/node_modules/webtorrent/lib/torrent.js:1471:23)
    at /usr/local/lib/node_modules/webtorrent-cli/node_modules/webtorrent/lib/torrent.js:1423:14
    at /usr/local/lib/node_modules/webtorrent-cli/node_modules/simple-sha1/index.js:6:5
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by feross** [comment]
Yes, thanks for the report. I forgot to merge the PR I sent to webtorrent-cli: https://github.com/feross/webtorrent-cli/pull/6

**Comment by feross** [comment]
Released as webtorrent-cli 1.1.3
